inc/field.class.php

```diff
@@ -218,17 +194,32 @@ public function getFieldTypeName() {
       return strtolower($matches[1]);
    }
 
-   public function getUsedParameters() {
+   public function getEmptyParameters() {
```

Comment:
rename method to distinguish instanciation of parameters and instanciation + fill from DB